File: Basic/src/Exercicios/Ex14.java
package Exercicios;

import java.text.NumberFormat;
import java.util.Scanner;

public class Ex14 {
    public static void main(String[] args) {
        System.out.println("Digite a quantidade de KM rodados");
        Scanner kms = new Scanner(System.in);
        double qtd_kms = kms.nextDouble();

        System.out.println("Digite a quantidade de dias alugadas");
        Scanner alugada = new Scanner(System.in);
        int dias_alugadas = alugada.nextInt();
        alugada.close();

        double preco_total = 90*dias_alugadas + 0.20*qtd_kms;
        System.out.println("O valor total a ser pago será "+ NumberFormat.getCurrencyInstance().format(preco_total));


    }
}

File: Algoritmos Python/Ex14.py
qtd_kms = float(input('Digite a quantidade de KM rodados'))
dias_alugadas = int(input('Digite a quantidade de dias alugados'))
preco_total = 90*dias_alugadas + 0.20&qtd_kms
print(f'O valor total a ser pago será R${preco_total:.2f}')

# Entrada
# A = lista1, lista2, lista3
# n = 3
# x = qtd_kms, dias_alugadas, preco_total
# Saida
# 1 Ajustar a reposta para NOT FOUND
# 2 Para cada indice i em A:
#   Se A[i] = x de cada lista e se lista1[i]*0.20 + lista2[i]*90 = lista3[i], ajustar i como resposta
# 3 Retornar a resposta como saida.